### DISCUSSION TITLE
Edit Calculation/Quotes Panel

### DISCUSSION THREAD

**Discussion by espe808** [YetiForceCompany/YetiForceCRM#14775] [How to]
Hello,
I would to edit this table with custom fields (with also calculated fields with immediate results).
Then I would also to export for selected row of this table the PDF template (like happens for Contacts/Accounts/Quotes...)

![120096752-855bb100-c12d-11eb-9827-201eb4fdccc9 (1)](https://user-images.githubusercontent.com/85027316/120771770-e44f6a80-c51f-11eb-8cc4-dd45165e362b.png)

How I could proceed?

**Comment by noelhunter** [comment]
Have you looked in Software configuration/Standard modules/Edit fields then select Quotes, and Managing and Advanced Block?
